ansible/deploy-measurements.yml

```diff
@@ -20,11 +20,11 @@
       name: oomsm-beta
```

Comment:
`letsencrypt_domains` is not updated.

Actually, changing hostname in inventory and hosts does not matter that much, but chaning `letsencrypt_domains` matters as it's used for SSL cert.

Reply:
Done